### ISSUE TITLE
AutoDMG does not recognize Big Sur 11.0 installer.app

### ISSUE THREAD

**Issue by MrMacintoshBlog** [MagerValp/AutoDMG#254]
## Description of issue

AutoDMG 1.10b does not recognize the new Big Sur Beta installer.app

## Expected behavior

Recognized installer

## Actual behavior

AutoDMG v1.10b 587 does not recognize the Install macOS Beta installer.

## Steps to reproduce

Downloaded 11.0 Big Sur from installinstallmacos.

Mount dmg > Drag Install macOS Beta to AutoDMG

AutoDMG does not recognize the installer.

## Log file

No log

**Comment by MagerValp** [comment]
Yep. OS installs are now signed APFS snapshots, and there's no obvious way of installing them to a dmg. I'm guessing that this is the end of the line.

**Comment by TimothyRLaMora734** [comment]
If you use an app called Create Install Media you can make a bootable usb installer https://mac.softpedia.com/get/System-Utilities/Create-Install-Media.shtml

**Comment by jfiliceatcsumb** [comment]
Check it...https://github.com/rtrouton/create_macos_vm_install_dmg

**Comment by flammable** [comment]
Copying the macOS installer into a DMG isn't the same thing as installing the macOS into a DMG. Neither of those tools are viable replacements for AutoDMG.

**Comment by eightnoneone** [comment]
Would it be possible to boot a brand new Mac into target disk mode and then copy the disk to an image?

**Comment by sln-guy** [comment]
Thanks for all the work on AutoDMG. It was a priceless tool for as long as Apple allowed it to work. 

Saved us all many hours, so for that thank you.
